xml/System.Runtime.CompilerServices/RuntimeHelpers.xml

```diff
@@ -767,9 +767,9 @@ This method is intended for compiler use rather than use directly in code. `T` m
       <Parameters />
       <Docs>
         <typeparam name="T">The type.</typeparam>
-        <summary>Returns a value that indicates whether the specified type is a reference type or a value type that contains references.</summary>
+        <summary>Returns a value that indicates whether the specified type is a reference type, a value type that contains references, or ByRefLike with references or by-refs.</summary>
         <returns>
-          <see langword="true" /> if the given type is reference type or value type that contains references; otherwise, <see langword="false" />.</returns>
+          <see langword="true" /> if the given type is a reference type, a value type that contains references, or ByRefLike with references or by-refs; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if the given type is a reference type or a value type that contains references or by-refs; otherwise, <see langword="false" />.</returns>
```